docs/api/ref/api-v3.yml

```diff
@@ -20,7 +20,7 @@ servers:
   - url: 'https://world.openfoodfacts.org'
     description: prod
   - description: dev
-    url: 'https://off:off@world.openfoodfacts.net'
+    url: 'https://@world.openfoodfacts.net'
```

Comment:
you have to remove the '@' (also in the other places)

Reply:
https://developer.mozilla.org/fr/docs/Web/HTTP/Authentication#acc%C3%A8s_avec_identifiants_dans_lurl